modules/swagger-codegen/src/main/resources/TypeScript-Fetch/configuration.mustache

```diff
@@ -0,0 +1,6 @@
+export class Configuration {
+    apiKey: string;
```

Comment:
@tht13 just a second thought that we should declare `apiKey` as dictionary instead of string to support multiple API keys. Other generators are already using dictionary/hash/map to store `apiKey`.

Later we'll need to fix the same for typescript-angular2.